elastic/kibana
# [Synthetics] only check for allowed status on ess

## Summary

Resolves https://github.com/elastic/kibana/issues/139210

This PR checks if a deployment is considered ESS by checking for the `isCloud` boolean, as well as the presence of the service manifest url. 

### Testing regression

**Please note**:  Testing was not performed in ECE for this PR. It's difficult to test in ECE with a non-public Kibana image. Once this PR makes it into the public snapshot registry for Kibana, testing can be performed in ECE. 

In order to test regression against ESS, I've created https://ece-pr-fix.kb.us-west2.gcp.elastic-cloud.com:9243/app

1. Login with credentials https://p.elstc.co/paste/WQTwcxne#g5DoRdjdsgsAl+C6vrLhbytzqOvTkRESRpOELxOgbBg
2. Navigate to Uptime monitor management
3. Confirm that the request access prompt is visible

### Testing progression

Once this PR makes it into the public snapshot, testing can be done via ecetest. https://github.com/elastic/observability-dev/tree/main/environments/ecetest
